src/components/who-is-online/types.ts

```diff
@@ -10,6 +11,16 @@ export enum Position {
 export interface Participant extends GeneralParticipant {
   slotIndex: number;
   isLocal?: boolean;
+  joinedPresence?: boolean | undefined;
+  isPrivate?: boolean;
 }
 
 export type WhoIsOnlinePosition = Position | `${Position}` | string;
+
+export type Data = {
```

Comment:
What's data?

Reply:
Just a subset of Participant. I changed the code to use Participant type instead